src/registry/resolve.rs

```diff
@@ -78,7 +65,10 @@ pub(crate) fn command(
     logger: impl Logger + Sync + Clone,
     cache: &Cache,
     args: &RegistryResolveArgs,
-) -> Result<(), DiagnosticMessages> {
+) -> Result<ExitDirectives, DiagnosticMessages> {
```

Comment:
Now this command can only display the resolved registry. See `weaver schema resolve` to display the resolved telemetry schema (this part is still experimental).